src/tests/Microsoft.PowerFx.Interpreter.Tests/PadUntypedObjectTests3.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class PadUntypedObjectTests3
+    {
+        public class PadTests3
+        {
+            [Fact]
+            public void PadUntypedObjectTest3()
```

Comment:
nit, 
// Test an untyped object that allows both index and property lookup. 
rename "TestArrayAndObject"